### PR TITLE
Front end/fix global theme

### PR DESCRIPTION
**Pull request by ebauss**

### CODE REVIEW

File: dna-frontend/src/styles.scss [hunk 1]
```diff
@@ -29,25 +29,7 @@
...
-  --tui-padding-m: 2rem;
-  --tui-padding-l: 3rem;
-  --tui-radius-xs: 0.25rem;
-  --tui-radius-s: 0.5rem;
-  --tui-radius-m: 0.75rem;
-  --tui-radius-l: 1rem;
-  --margin-xs: 0.5rem;
-  --margin-s: 1rem;
-  --margin-m: 2rem;
-  --margin-l: 3rem;
```
Comment (nng32) [inline dna-frontend/src/styles.scss:50]: These classes were used by the login page
image

**Review by nng32 — CHANGES_REQUESTED** [review]
Auth page spacing got messed up

File: dna-frontend/src/styles.scss (hunk 1 above)
Comment (nng32) [reply dna-frontend/src/styles.scss:50]: Signup as well
image

Comment (ebauss) [reply dna-frontend/src/styles.scss:50]: rip

**Review by nng32 — APPROVED** [review]
lgtm!

**Pull request merged**